docs/examples/Configuring_QCoDeS.ipynb

```diff
@@ -17,16 +17,17 @@
    "source": [
     "# QCoDeS config\n",
     "\n",
-    "The Qcodes config module uses JSON files to store QCoDeS configuration.\n",
+    "The QCoDeS config module uses JSON files to store QCoDeS configuration.\n",
     "\n",
-    "The config file controls various options to QCoDeS such as the default location where data is stored and logging level of debug output. A default configuration is shipped with QCoDeS. As we will see below you can overwrite this configuration in several different ways to customize the configuration. For instance you may want to change the location of the `qcodes.dataset` experiments db. In the example below you should see that by default it is stored in `~/experiments.db` (`~` is shorthand for the users home folder) but you can see that in the current config I have changed it to be stored inside a sub folder of my home folder. \n"
+    "The config file controls various options to QCoDeS such as the default path and name of the database in which your data is stored and logging level of the debug output. QCoDeS is shipped with a default configuration. As we shall see, you can overwrite these default values in several different ways to customize the configuration. In particular, you may want to change the path of your database which is initialised by the `qcodes.dataset` with the default name `experiments.db`. By default the path of your database is `~/experiments.db` (here, `~` stands for the path of the user's home directory). In the following example, I have changed the default path of my database, flagged with the key `db_location`, \n",
```

Comment:
to my taste, the "the default path of my database, flagged with the key `db_location`" part is confusing, i think it can be written in a more clear way.

also "my" and "I" - i'm looking at `'c:\\\\Users\\\\jenielse\\\\mymainfolder\\\\experiments.db'` below, and does this mean that this notebook is written with Jens as a narrator in mind? :)

Reply:
Before it wan not clear in the text that which key in the dictionary actually corresponds to the location of the dbase.  I can rewrite it. 

I think Jens first wrote this example, so I decided to leave those path names as it is. I believe that It will make no difference.

Reply:
> Before it was not clear in the text that which key in the dictionary actually corresponds to the location of the dbase

ah. ok, but the thing is that the schema of the config file is also added to the docs, and the schema explains all the deafault keys, so perhaps it's more efficient to refer to it?

Reply:
In this case, I believe, this should stay as in the cell immediately after, the core dictionary is provided. I will just change "flagged with" with "represented by". A general reference to the schema is not necessary, IMHO.

Reply:
ok, btw this is where the config is completely descibed (what i referred to as "schema") http://qcodes.github.io/Qcodes/user/configuration.html#default-config